config/samples/netbox_v1_prefixclaim_customfields.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: netbox.dev/v1
+kind: PrefixClaim
+metadata:
+  labels:
+    app.kubernetes.io/name: netbox-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: prefixclaim-customfields-sample
+spec:
+  tenant: "MY_TENANT" # Use the `name` value instead of the `slug` value
+  site: "DM-Akron" # Use the `name` value instead of the `slug` value
+  description: "some description"
+  comments: "your comments"
+  preserveInNetbox: true
+  prefixLength: "/31"
+  parentPrefixSelector: # The keys and values are case-sensitive
+    environment: "Production"
+    poolName: "Pool 1"
```

Comment:
A bit confused here. The custom field name is `poolName`. Shouldn't the filter in here then be `cf_poolName`? Or how do you distinguish custom fields from built in fields? I think it would make sense to be consistent with the netbox filtering https://demo.netbox.dev/static/docs/rest-api/filtering/#filtering-by-custom-field

Reply:
I manually filter out `tenant` and `site`, so we don't force the user to build the netbox API query string in the CRs. 

Should we have the user enter the prefix `cf_` when adding the entries to save us some implementation hassle, or, we handle this for the user, which looks nicer on the UX standpoint.  What do you think?

Reply:
I'm fine both ways. If we handle `cf_` for the user it's one one side nice but on the other side also confusing. Depends a bit whether we want to stick to the UI (handle it for the user) or API (the user has to specify `cf_` himself). wdyt @bruelea ?

Reply:
For spec.customFields the `cf_` is handled for the use, so we could do the same for parentPrefixSelector.

Reply:
Thanks @jstudler and @bruelea for the discussion. Closing as we have reached the conclusion to keep the existing behavior.